checkup.sh

```diff
@@ -0,0 +1,61 @@
+
+#!/bin/bash
+
+nodev=$(node -v);
+funcv=$(func -v);
+echo "Your node version is $nodev."
+echo "Your Azure Functions Core Tools version is $funcv."
+
+echo "--------------------------------";
+echo "Checking if node is installed...";
+echo "--------------------------------";
+if which node > /dev/null
+then
+    echo "Node is installed, skipping...";
+else
+    echo "Node is not installed.";
```

Comment:
Would students need to restart their terminal to see that node is infact installed? I know that sometimes that needs to happen. Maybe add a line saying "If you've installed `node` but this script is still saying you haven't, please restart your terminal and run the script again"

Reply:
We should put that into the output at the very beginning